perfkitbenchmarker/openstack/os_virtual_machine.py

```diff
@@ -0,0 +1,155 @@
+import logging
+import random
+import string
+import time
+
+from perfkitbenchmarker import virtual_machine, linux_virtual_machine
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.openstack import utils as os_utils, os_disk
+
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('os_auth_url', 'http://localhost:5000',
+                    'This determine url to Keystone authenticate service.'
+                    'It is require to discovery other OpenStack services URLs')
+
+flags.DEFINE_string('os_username', 'admin',
+                    'OpenStack login')
+
+flags.DEFINE_string('os_passwd', 'admin',
```

Comment:
Requiring passwords on the command line isn't ideal. Alternatively, consider passing these via a configuration file or through environment variables? Looks like the Nova CLI tool uses `OS_PASSWORD` for this purpose.